ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/QueryResponseHandler.java

```diff
@@ -62,7 +59,8 @@ private void handleArgs(final Buffer buff) {
 
     final QueryResponseMetadata queryResponseMetadata;
     try {
-      queryResponseMetadata = JSON_MAPPER.readValue(buff.getBytes(), QueryResponseMetadata.class);
+      queryResponseMetadata = ApiJsonMapper.INSTANCE.get()
```

Comment:
this is not matching with 7.3.x . Either we can keep the older version with JsonMapper or we can handle it the way 7.3.x does.

Reply:
I checked. Internally both are calling same APIs. Will keep it as is.